ui/src/assets/i18n/en.json

```diff
@@ -1023,5 +1028,14 @@
   "workflow_wizard_select_repo_loading": "Loading repositories...",
   "workflow_wizard_repo_analyse" : "Analyzing repository...",
   "workflow_wizard_select_repo_man_add" : "Or add a repository manager",
-  "wizard_repo_files" : "Files found: "
+  "wizard_repo_files" : "Files found: ",
+
+  "workflow_template_add_title": "Create workflow template",
+  "workflow_template_edit_title": "Edit workflow template",
+  "workflow_template_created": "Workflow template created",
+  "workflow_template_saved": "Workflow template saved",
+  "workflow_template_deleted": "Modèle de workflow deleted",
```

Comment:
Worflow template deleted (it's the 'en' file)

Reply:
done